pkg/disttask/framework/storage/util.go

```diff
@@ -104,11 +104,27 @@ func GetSubtaskEndTimeForTest(ctx context.Context, stm *TaskManager, subtaskID i
 	return time.Time{}, nil
 }
 
-// PrintSubtaskInfo log the subtask info by taskKey. Only used for UT.
-func (stm *TaskManager) PrintSubtaskInfo(ctx context.Context, taskID int64) {
-	rs, _ := stm.executeSQLWithNewSession(ctx,
+// GetSubtaskNodesForTest gets subtasks running nodes for one task for test.
+func GetSubtaskNodesForTest(ctx context.Context, mgr *TaskManager, taskID int64) ([]string, error) {
+	rs, err := mgr.executeSQLWithNewSession(ctx,
+		`select exec_id from mysql.tidb_background_subtask where task_key=%? group by exec_id`, taskID)
```

Comment:
It is not necessary to use `group by`, maybe `select distinct(exec_id)` is enough.

Reply:
https://github.com/pingcap/tidb/pull/50115/commits/1cbf435edd22be486cbeffda9492ff99efe03d76